### PR TITLE
fix(picker): ensure picker doesn't crash if its value is set to null

### PR DESCRIPTION
**Pull request by jensgustafsson**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/picker/picker.tsx [hunk 1]
```diff
@@ -203,6 +203,8 @@ export class Picker {
     @Watch('value')
     protected onChangeValue(newValue = [], oldValue = []) {
         this.chips = this.createChips(this.value);
+        newValue = newValue || [];
```
Comment (Befkadu1) [inline src/components/picker/picker.tsx:206]: newValue = [] and oldValue = [] by default. Right? I think we don't have to set it again.

**Comment by Befkadu1** [comment]
Still picker crashes when portalElement=null at line 378. It crashes here, `parent.contains(element)` when `parent` is null in dom.ts file. We should make sure that `parent` not null.

Screenshot 2020-04-29 at 17 44 10

File: src/components/picker/picker.tsx (hunk 1 above)
Comment (jensgustafsson) [reply src/components/picker/picker.tsx:206]: Unfortunately not. If `null` is provided as an argument then `newValue` will be `null` and not `[]`

**Comment by jensgustafsson** [comment]
> Still picker crashes when `portalElement=null` at `isDescendant(element as HTMLElement, portalElement as HTMLElement)` around line 378 . It crashes here, `parent.contains(element)` when `parent` is null in dom.ts file. We should make sure that `parent` is not null.
> 
> Screenshot 2020-04-29 at 17 44 10

Maybe you can show me remotely tomorrow?

**Comment by Befkadu1** [comment]
> > Still picker crashes when `portalElement=null` at `isDescendant(element as HTMLElement, portalElement as HTMLElement)` around line 378 . It crashes here, `parent.contains(element)` when `parent` is null in dom.ts file. We should make sure that `parent` is not null.
> > Screenshot 2020-04-29 at 17 44 10
> 
> Maybe you can show me remotely tomorrow?

Ok

**Pull request merged**